app/views/partials/_leftbar.html.haml

```diff
@@ -24,8 +24,13 @@
       %i.fa.fa-beer.fa-2x.fa-fw>
       Beers
   %li
-    %a#links.green-text.text-lighten-2{:href => "#"}
+    %a#links.green-text.text-lighten-2{:href => url_for(controller: "users", id: current_user, action: "edit")}
```

Comment:
After ruby `1.9` this hash rocket/fat arrow syntax can be updated as `href: ` instead of `:href =>`